bin/pocs_shell

```diff
@@ -74,6 +74,13 @@ class PocsShell(Cmd):
 
         return self.pocs.is_safe()
 
+    def do_display_config(self, *arg):
+        """ Display the config file for POCS """
+        if self.pocs.config is None:
```

Comment:
What if self.pocs is not set (i.e. setup_pocs has not been run)? Seems like it would throw an exception.

Reply:
@jermainegug Can you check on the above?